### DISCUSSION TITLE
Transferring Domain Rights from My Project to My Friend's Project

### DISCUSSION THREAD

**Discussion by qsig-main** [vercel/community#3706] [Feature Question] [answered]
### Question

My personal Vercel account is currently hosting a project that is using a registered/owned domain name. The linkage between my project and the domain name is well-established (DNS custom NameServer records are set up correctly).

My friend's Vercel account is hosting a project and is using the default Vercel domain name (https://[insert project name]-xxxxxx-[friend's username].vercel.app/team). 

I want to give up the rights to the domain name and let my friend us it for his project. In other words, I want to forfeit my domain name from my current project. What steps should me and my friend take?

(The domain name registrar is HostPapa)

**Comment by amyegan** [comment, marked as answer]
Good question! You can transfer a domain to another Vercel user or team by following these steps: https://vercel.com/docs/projects/domains/transfer-your-domain#transfer-a-domain-to-another-vercel-user-or-team
